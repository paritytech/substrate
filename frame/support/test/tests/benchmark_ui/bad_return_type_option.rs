use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;

#[benchmarks]
mod benches {
	use super::*;

	#[benchmark]
	fn bench() -> Option<BenchmarkError> {
		#[block]
		{}
		assert_eq!(2 + 2, 4);
		None
	}
}

fn main() {}
