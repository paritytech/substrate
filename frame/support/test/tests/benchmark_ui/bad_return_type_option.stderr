error: expected `Result`
  --> tests/benchmark_ui/bad_return_type_option.rs:10:16
   |
10 |     fn bench() -> Option<BenchmarkError> {
   |                   ^^^^^^
